### DISCUSSION TITLE
I need to know which tracks are hidden and which tracks are visible

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#1247] [Q&A] [answered]
I need to know which tracks are hidden and which tracks are visible, but it seems that Alphatab does not record this information.

![未标题-1](https://github.com/CoderLine/alphaTab/assets/17884664/503fd045-9692-46f5-ba4f-913f993f5598)

**Comment by Danielku15** [comment, marked as answer]
This is true, we currently provide no way to access this information. 

In GP6-8 the currently selected track in single track and multi-track setups is part of the `PartConfiguration` https://github.com/CoderLine/alphaTab/blob/develop/src/importer/PartConfiguration.ts

Internally the PartConfiguration is rather a set of special "views" to the score. It technically could allow configuring multiple combinations of tracks and whether standard/tablature etc. information is shown but the Guitar Pro UI doesn't allow these setups. They have 1 multi track view and one view for each individual track. 

For GP3-5 am not sure if such information about the currently shown tracks exists. 

I added work item into the backlog/roadmap here: https://github.com/orgs/CoderLine/projects/12/views/1?pane=issue&itemId=38121116

**Reply by Danielku15** [reply]
@goodgame365 The feature was implemented as part of https://github.com/CoderLine/alphaTab/issues/1486 and will be in the next 1.4 nightly build.
